Source/inv.cpp

```diff
@@ -330,14 +330,9 @@ void DrawInvBelt(const CelOutputBuffer &out)
 		if (AllItemsList[myPlayer.SpdList[i].IDidx].iUsable
 		    && myPlayer.SpdList[i]._iStatFlag
 		    && myPlayer.SpdList[i]._itype != ITYPE_GOLD) {
-			sprintf(tempstr, "%i", i + 1);
-			SDL_Rect rect {
-				InvRect[i + SLOTXY_BELT_FIRST].x + PANEL_X + InventorySlotSizeInPixels.width - GetLineWidth(tempstr),
-				InvRect[i + SLOTXY_BELT_FIRST].y + PANEL_Y - 1,
-				0,
-				0
-			};
-			DrawString(out, tempstr, rect, UIS_SILVER);
+			snprintf(tempstr, sizeof(tempstr) / sizeof(*tempstr), "%i", i + 1);
+			// FIXME: Replace this with Rectangle(Point, Size) once https://github.com/diasurgical/devilutionX/pull/2131 is merged

```

Comment:
It's a bit unclear to me if this has already happened or not, could you point me to the specific commit where the dependant change happens?

Reply:
That comment is out of date, I should've removed it 😅

Reply:
Alright, that helps motivate me to review a bit more :)